system-int-test/test/cmr/system_int_test/search/facets/collection_facets_v2_search_test.clj

```diff
@@ -176,14 +177,23 @@
   (testing "Facets size applied for facets, with selecting facet that exists, but outside of the facets size range."
     (is (= fr/expected-v2-facets-apply-links-with-selecting-facet-outside-of-facets-size
            (search-and-return-v2-facets {:facets-size {:platforms 1}
-                                         :platforms-h {:0 {:short-name "diadem-1D"}}}))))
+                                         :platforms-h {:0 {:basis "Space-based Platforms"
```

Comment:
there is a helper function for defining this platform at the top of the function

Reply:
I used it.